inc/RequestParser.hpp

```diff
@@ -0,0 +1,56 @@
+#pragma once
+
+/* ====== LIBRARIES ====== */
+
+#include "HTTPRequest.hpp"
+#include "StatusCode.hpp"
+#include "error.hpp"
+#include "utilities.hpp"
+#include <iostream>
+#include <map>
+#include <sstream>
+#include <stdexcept>
+#include <stdint.h>
+#include <vector>
+
+/* ====== DEFINITIONS ====== */
+
+const int versionPrefixLength = 5; // HTTP/ -> 5 chars
+const int getLength = 3; // GET -> 3 chars
+const int postLength = 4; // POST -> 4 chars
+const int deleteLength = 6; // DELETE -> 6 chars
+const int decimalBase = 10;
```

Comment:
Do these need to be global? Could just be defined in the functions where they are used.

Reply:
Moved those which are used in only one function to the respective function.